client/lib/abtest/active-tests.js

```diff
@@ -30,10 +30,11 @@ module.exports = {
 		defaultVariation: 'originalA'
 	},
 	freeTrials: {
-		datestamp: '20160120',
+		datestamp: '20160322',
 		variations: {
-			notOffered: 90,
-			offered: 10
+			notOffered: 80,
+			addedInSignup: 10,
+			offeredOnThankYouPage: 10
```

Comment:
Should we make this a higher percentage, given that many less people will end up in this bucket?